zebra/interface.c

```diff
@@ -1156,6 +1156,94 @@ static const char *zebra_ziftype_2str(zebra_iftype_t zif_type)
 	}
 }
 
+/* Interface's brief information print out to vty interface. */
+static void ifs_dump_brief_vty(struct vty *vty, struct vrf *vrf)
+{
+	struct connected *connected;
+	struct listnode *node;
+	struct route_node *rn;
+	struct zebra_if *zebra_if;
+	struct prefix *p;
+	struct interface *ifp;
+	bool print_header = true;
+
+	FOR_ALL_INTERFACES (vrf, ifp) {
+		bool is_global_v6 = false;
+		char global_pfx[PREFIX_STRLEN] = {0};
+		char ll_pfx[PREFIX_STRLEN] = {0};
+		bool break_loop = false;
+		char buf[PREFIX_STRLEN] = {0};
+
+		if (print_header) {
+			vty_out(vty, "%-8s%-16s%-16s%-20s%-51s\n", "status",
```

Comment:
@nitinsoniism There's also code to generate ASCII tables in lib that nobody ever uses, it makes me sad... `lib/termtable.c` will do all the formatting for you...

Reply:
@qlyoung trust me I badly wanted to use termtable after I came to know about this from @donaldsharp :). I also got the same comment above by @rwestphal. Here is the response I posted -  

_We did start with using the termtable but initially, the function which does the job of printing the actual output was getting called every time for each interface. This made it impossible to use termtable. We continued and added the code the way it is right now. Later because of some reason we had to move the interface iteration code also inside the added function and it was a good reason for using termtable but since we had already written the code and all the surrounding code was using the same way of display, we decided to not use termtable._

I might be changing a lot of code after some of the comments from @eqvinox and if possible, I will incorporate termtable.

Reply:
- Name is now before the status.

- Column headers are properly capitalized.

- Got rid of trailing space characters.